### PR TITLE
feat: (IAC-486) Update default cadence version value

### PR DESCRIPTION
**Pull request by riragh**
# Changes:
The current default cadence version value is set to LTS "2020.1" which is out of support now, hence updated the cadence version value to latest version LTS "2022.09"

# Tests:
Verified following scenarios, please see test details in the internal ticket.
- With cadence version not specified in ansible-vars.yaml, the default value  LTS "2022.1" is picked - DO enabled
- With cadence version not specified in ansible-vars.yaml, the default value  LTS "2022.1" is picked - DO disabled
- Specifying any other cadence value does not cause any issue

### CODE REVIEW

**Review by thpang — APPROVED** [review]

File: roles/vdm/defaults/main.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ V4_CFG_LICENSE: null
 V4_CFG_DEPLOYMENT_ASSETS: null
 V4_CFG_ORDER_NUMBER: null
 V4_CFG_CADENCE_NAME: lts
-V4_CFG_CADENCE_VERSION: "2020.1"
+V4_CFG_CADENCE_VERSION: "2022.09"
```
[the remark this answers is not in the document]
Comment (riragh) [reply roles/vdm/defaults/main.yaml:10]: I previously picked the version LTS 2022.1, however the support kubernetes version range for that LTS is 1.20.x - 1.22.x, (see [here](https://go.documentation.sas.com/doc/en/sasadmincdc/v_027/itopssr/n1ika6zxghgsoqn1mq4bck9dx695.htm#n0mhb21pl07ohgn1hj4zsgfeq8hj)) which ran into issues when running with DO. So updated it to the most latest LTS version "2022.09" which support the kubernetes version range 1.21.x - 1.24.x .

**Comment by jarpat** [comment]
Small update here
https://github.com/sassoftware/viya4-deployment/blob/IAC-486/docs/CONFIG-VARS.md?plain=1#L124
2020.1 > 2022.09

File: roles/vdm/defaults/main.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply roles/vdm/defaults/main.yaml:10]: ok, makes sense

**Comment by riragh** [comment]
> Small update here https://github.com/sassoftware/viya4-deployment/blob/IAC-486/docs/CONFIG-VARS.md?plain=1#L124 2020.1 > 2022.09

Good Catch. I'll update it. I noticed `V4_CFG_CADENCE_VERSION` is present twice in that table, I'll remove the duplicate as well.

**Review by dhoucgitter — APPROVED** [review]

**Pull request merged**